dbt/models/qc/docs.md

```diff
@@ -42,3 +42,73 @@ Pulled from the following Inquire queries:
 
 - `FP Checklist - Negative ASMT Values` (#1569) - Will C.
 {% enddocs %}
+
+# vw_iasworld_sales_null_values
+
+{% docs view_vw_iasworld_sales_null_values %}
+Test that deed, buyer, seller, and price `iasworld.sales` columns are not null.
+
```

Comment:
**nitpick:** If the description is just a single line, then you can remove the newline here:

```suggestion
```